packages/myst-to-typst/src/container.ts

```diff
@@ -41,40 +49,41 @@ export const containerHandler: Handler = (node, state) => {
     });
     return;
   }
-  state.useMacro('#show figure: set block(breakable: true)');
+
+  state.ensureNewLine();
+  state.write('#show figure: set block(breakable: true)');
```

Comment:
I wrote this directly into the document - the `macro` was not importing across `.typ` files. Still maybe should just be on the template.

Reply:
This also is a little overzealous - looks like a figure at the end of a page that could easily fit on the next page will remain in place and be broken...